src/react/projects/spark-react/src/SprkStepper/components/SprkStepperSlider/SprkStepperSlider.js

```diff
@@ -1,8 +1,31 @@
 import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 class SprkStepperSlider extends Component {
+  constructor(props) {
+    super(props);
+
+    this.sliderRef = React.createRef();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.sliderStyle.top != this.props.sliderStyle.top){
+      this.sliderRef.current.classList.remove('sprk-c-Stepper__slider--active');
```

Comment:
can this cause/effect relationship be unit tested? Aka, change `top`, observe `--active` flicker? Not sure if that adds value now, but would stop it from changing in the future.

Reply:
i think componentWIllReceiveProps is deprecated also